compiler/rustc_codegen_llvm/src/debuginfo/metadata/enums/cpp_like.rs

```diff
@@ -507,8 +803,38 @@ fn variant_union_field_name(variant_index: VariantIdx) -> Cow<'static, str> {
         "variant15",
     ];
 
+    if Some(variant_index) == dataful_variant_index {
+        return Cow::from("variant_fallback");
```

Comment:
What is the meaning of `variant_fallback`?  Is this something a debug visualizer needs to handle?

Reply:
Yes, it's used for niche-layout enums, so it's very common. I'll add some documentation.